vault-plugins/flant_iam/usecase/featureflag.go

```diff
@@ -0,0 +1,175 @@
+package usecase
+
+import (
+	"github.com/flant/negentropy/vault-plugins/flant_iam/model"
+	"github.com/flant/negentropy/vault-plugins/shared/io"
+)
+
+// TenantFeatureFlagger manager feature flags for tenants
+type TenantFeatureFlagger struct {
```

Comment:
Probably TennantFeatureFlagService - better name?

Reply:
Let's imagine the usage:

```go
tenant, err := usecase.Tenants(tx).GetByID(id)
tenant, err := usecase.TenantsService(tx).GetByID(id)
tenant, err := usecase.TenantsManager(tx).GetByID(id)
tenant, err := usecase.TenantsManager(tx).GetByID(id)

roles, err := usecase.TennantFeatureFlags(tx).AvailableRoles()
roles, err := usecase.TennantFeatureFlagger(tx).AvailableRoles()
roles, err := usecase.TennantFeatureFlagService(tx).AvailableRoles()
roles, err := usecase.TennantFeatureFlagManager(tx).AvailableRoles()
```

To me, words like "service", "manager", etc. do not add clarity. We have "usecase" prefix. Isn't that enough?

```go
err := usecase.RoleBindings(tx).Update(rb)
err := usecase.Groups(tx).Create(group)
err := usecase.ServiceAccountPasswords(tx, said).Create(password)
```